OrbitQt/orbitmainwindow.cpp

```diff
@@ -1060,16 +1091,27 @@ void OrbitMainWindow::SetTarget(const orbit_qt::StadiaTarget& target) {
   GOrbitApp->SetGrpcChannel(connection->GetGrpcChannel());
   GOrbitApp->SetProcessManager(target.GetProcessManager());
   GOrbitApp->SetProcess(target.GetProcess());
+
+  std::string target_label_text = absl::StrFormat(
+      "%s @ %s", target.GetProcess()->name(), connection->GetInstance().display_name.toStdString());
+  target_label_->setText(QString::fromStdString(target_label_text));
+  target_label_->setStyleSheet(QString("color: %1").arg(kGreenColor));
 }
 
 void OrbitMainWindow::SetTarget(const orbit_qt::LocalTarget& target) {
   const orbit_qt::LocalConnection* connection = target.GetConnection();
   GOrbitApp->SetGrpcChannel(connection->GetGrpcChannel());
   GOrbitApp->SetProcessManager(target.GetProcessManager());
   GOrbitApp->SetProcess(target.GetProcess());
+
+  std::string target_label_text = absl::StrFormat("%s @ localhost", target.GetProcess()->name());
+  target_label_->setText(QString::fromStdString(target_label_text));
+  target_label_->setStyleSheet(QString("color: %1").arg(kGreenColor));
 }
 
-void OrbitMainWindow::SetTarget(const orbit_qt::FileTarget& /*target*/) {}
+void OrbitMainWindow::SetTarget(const orbit_qt::FileTarget& target) {
+  target_label_->setText(QString::fromStdString(target.GetCaptureFilePath().filename().string()));
```

Comment:
This should probably also set the target label color and not rely on it being set in the beginning to be consistent with the in- and out-conditions of the other `SetTarget` methods.

Reply:
Fixed. @beckerhe PTAL